RQ_INTEGRATION_README.md

```
@@ -0,0 +1,354 @@
+
+```bash
+# Access Redis directly
+docker-compose exec redis redis-cli
+
+# List all keys
+KEYS *
+
+# Check queue length
+LLEN rq:queue:facial_recognition
+
+# Monitor Redis operations
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)